dataactvalidator/scripts/load_object.py

```diff
@@ -0,0 +1,55 @@
+import os
+import logging
+
+import pandas as pd
+import boto
+
+from dataactcore.config import CONFIG_BROKER
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.logging import configure_logging
+from dataactcore.models.domainModels import ObjectClass
+from dataactvalidator.health_check import create_app
+from dataactvalidator.scripts.loaderUtils import clean_data, insert_dataframe
+
+logger = logging.getLogger(__name__)
+
+
+def load_object_class(base_path):
+
+    """
+    This function loads Object classes into the database
+        Args:
+            base_path : directory that contains the domain values files.
```

Comment:
Please take another look at the StyleGuide.md for this comment formatting. Specifically, the `Function comments` section